example/image_compiler/web/main.dart

```diff
@@ -0,0 +1,7 @@
+import 'dart:html';
+import 'pics/images.dart' as pics;
+
+void main() {
+  (querySelector('#inlined_image') as ImageElement).src = pics.icon;
+  querySelector('#output').text = 'Your Dart app is running.';
```

Comment:
I'd drop this, and would include a static text in the html: "The following image was inlined in Dart code:"